src/encoder.js

```diff
@@ -454,7 +456,9 @@ class Encoder {
       size += resultLength[i]
     }
 
-    var res = Buffer.allocUnsafe(size)
```

Comment:
This is probably where the perf differences is coming from.

There’s no equivalent to this outside of Node.js, so this change is moving us from a request for an already allocated piece of dirty memory to a new allocation of zero’d memory.